bin/cli.js

```diff
@@ -161,19 +179,22 @@ const loadConfigMiddleware = async options => {
       );
       process.exit(EXIT_CODES.ERROR);
     }
+    maybeValidateConfig();
     return;
   }
 
   // We need to load the config when options.config exists,
   // so that getAccountIdFromConfig() in injectAccountIdMiddleware reads from the right config
   if (options.config && fs.existsSync(options.config)) {
     const { config: configPath } = options;
-    await loadConfig(configPath, options);
+    loadConfig(configPath, options);
+    maybeValidateConfig();
```

Comment:
You could DRY this up a little and avoid calling `maybeValidateConfig` many times by switching to if/else blocks and removing the return calls.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1307/commits/c3fd8cc35cb4f896afc9f9af520107a6cd95732f